operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
```

Comment:
Because we are creating new types, we need to tell the remove playbook to delete these. This is required when Kiali is to be undeployed. So you need to add ConsoleLink to this list: https://github.com/kiali/kiali/blob/master/operator/roles/kiali-remove/tasks/main.yml#L164

Or, we may have to do something differently because the ConsoleLinks don't have easily hardcoded names (like "kiali" so you will have to delete them based on a query of the metadata labels (which is one reason why adding the metadata labels is needed - see below).

Reply:
Ok, will give it a try.

Reply:
Should be ok now.